app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -79,61 +71,9 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
     initialValues = formValues;
   }
 
-  const inPersonLocationPostOfficeSearchForm = inPersonFullAddressEntryEnabled
-    ? InPersonLocationFullAddressEntryPostOfficeSearchStep
-    : InPersonLocationPostOfficeSearchStep;
-
-  const inPersonSteps: FormStep[] =
-    inPersonURL === undefined
-      ? []
-      : ([
-          {
-            name: 'prepare',
-            form: InPersonPrepareStep,
-            title: t('in_person_proofing.headings.prepare'),
-          },
-          {
-            name: 'location',
-            form: inPersonLocationPostOfficeSearchForm,
-            title: t('in_person_proofing.headings.po_search.location'),
-          },
-          flowPath === 'hybrid' && {
-            name: 'switch_back',
-            form: InPersonSwitchBackStep,
-            title: t('in_person_proofing.headings.switch_back'),
-          },
-        ].filter(Boolean) as FormStep[]);
-
-  const steps: FormStep[] = submissionError
-    ? (
-        [
-          {
-            name: 'review',
-            form:
-              submissionError instanceof UploadFormEntriesError
-                ? withProps({
-                    remainingAttempts: submissionError.remainingAttempts,
-                    isFailedResult: submissionError.isFailedResult,
-                    isFailedDocType: submissionError.isFailedDocType,
-                    captureHints: submissionError.hints,
-                    pii: submissionError.pii,
-                    failedImageFingerprints: submissionError.failed_image_fingerprints,
-                  })(ReviewIssuesStep)
-                : ReviewIssuesStep,
-            title: t('errors.doc_auth.rate_limited_heading'),
-          },
-        ] as FormStep[]
-      ).concat(inPersonSteps)
-    : ([
-        {
-          name: 'documents',
-          form: DocumentsStep,
-          title: t('doc_auth.headings.document_capture'),
-        },
-      ].filter(Boolean) as FormStep[]);
-
+  const inPersonProofingStepNames = ['location', 'prepare', 'switch_back'];
```

Comment:
Since this value never changes, I'd suggest moving it somewhere toward the top of the file to avoid allocating a new array every time the component is rendered.

```ts
const IN_PERSON_PROOFING_STEP_NAMES = ['location', 'prepare', 'switch_back'];
```

Less important (especially since this code already existed), but we could also consider using a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) for this, which is a little more accurate in representing a set of unique values, and has some performance benefit (not that I expect performance to be much an issue here).

```ts
const IN_PERSON_PROOFING_STEP_NAMES = new Set(['location', 'prepare', 'switch_back']);
```